app/templates/scripts/requirejs/config.js

```diff
@@ -6,7 +6,7 @@ var require = {
         app: '../app/Resources/public/scripts/app',
         modules: '../app/Resources/public/scripts/modules',
         jquery: 'jquery/dist/jquery',
-        loglevel: 'loglevel/dist/loglevel.min'<% if (useFoundation) { %>,
+        loglevel: 'loglevel/dist/loglevel.min',<% if (useFoundation) { %>
```

Comment:
nice catch. 
The comma needs to be inside the conditional block so you won't hit an error when using something other than `foundation` or `bootstrap`